tests/auth_tests/test_hashers.py

```diff
@@ -209,6 +214,53 @@ def setter(password):
         finally:
             hasher.rounds = old_rounds
 
+    @skipUnless(argon2, "argon2-cffi not installed")
```

Comment:
Could you put these in their own test class so we only need skipUnless on the class?